src/virtual-machine.js

```diff
@@ -94,8 +94,8 @@ class VirtualMachine extends EventEmitter {
         this.runtime.on(Runtime.VISUAL_REPORT, visualReport => {
             this.emit(Runtime.VISUAL_REPORT, visualReport);
         });
-        this.runtime.on(Runtime.TARGETS_UPDATE, () => {
-            this.emitTargetsUpdate();
+        this.runtime.on(Runtime.TARGETS_UPDATE, emitProjectChanged => {
+            this.emitTargetsUpdate(emitProjectChanged);
```

Comment:
Is this ever true?

Reply:
Right now it isn't, but I don't know whether it's better to just assume always false here...

In a future pass we plan to completely separate out targets update and project changed, so we can fix this there then too. I'll file an actual issue for it.

Reply:
Filed in https://github.com/LLK/scratch-vm/issues/1964